src/pages/ReferencePage.vue

```diff
@@ -1,162 +1,276 @@
 <template>
   <div>
```

Comment:
let's replace this with b-container tag to standardize the theme with other pages